src/simplewallet/simplewallet.cpp

```diff
@@ -1838,25 +1911,28 @@ bool simple_wallet::show_incoming_transfers(const std::vector<std::string>& args
   {
     if (!filter || available != td.m_spent)
     {
+      if (m_wallet->get_current_subaddress_account() != td.m_subaddr_index.major || (!subaddr_indices.empty() && subaddr_indices.count(td.m_subaddr_index.minor) == 0))
+        continue;
```

Comment:
It seems very cumbersome otherwise, if you have to do 2N commands. Same as me not really liking the concept of implicit current account I guess. If you think it makes more sense this way I'm fine with it, though.